extraTs/patterns/diamond.ts

```diff
@@ -0,0 +1,22 @@
+export default function diamond(a) {
```

Comment:
return type of function and argument 'a' type is missing.

Reply:
Changes Done.